### PR TITLE
MAX_TWEET_LENGTH env implementation

### PR DESCRIPTION
**Pull request by onur-saf**
# Relates to:

# Risks

# Background

## What kind of change is this?

# Documentation changes needed?

# Testing

## Detailed testing steps

add MAX_TWEET_LENGTH env variable to .env file
start the agent with Twitter Client
check the limit is equal to MAX_TWEET_LENGTH value

### CODE REVIEW

**Review by monilpat — APPROVED** [review]
LGTM! Thanks for doing this :)

File: packages/client-twitter/src/environment.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { IAgentRuntime } from "@ai16z/eliza";
 import { z } from "zod";
 
+const DEFAULT_MAX_TWEET_LENGTH = 200;
```
Comment (oxSaturn) [inline packages/client-twitter/src/environment.ts:4]: Why change it to 200 when it's 280 in the original code?

File: packages/client-twitter/src/post.ts [hunk 2]
```diff
@@ -30,20 +30,21 @@ const twitterPostTemplate = `
 Write a 1-3 sentence post that is {{adjective}} about {{topic}} (without mentioning {{topic}} directly), from the perspective of {{agentName}}. Do not add commentary or acknowledge this request, just write the post.
 Your response should not contain any questions. Brief, concise statements only. The total character count MUST be less than 280. No emojis. Use \\n\\n (double spaces) between statements.`;
```
Comment (oxSaturn) [inline packages/client-twitter/src/post.ts:31]: Btw, there's another `280` here.

File: packages/client-twitter/src/environment.ts (hunk 1 above)
Comment (onur-saf) [reply packages/client-twitter/src/environment.ts]: ahh you're right it's updated

**Review by cygaar — APPROVED** [review]

**Pull request merged**